src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java

```diff
@@ -122,6 +112,14 @@ void hangingRequestDisconnectHandler(Object sender,
   }
 
 
+  public static boolean isLogAllWireBytes() {
+    return logAllWireBytes;
+  }
+
```

Comment:
maybe we should also keep track on parallelity. Calling the getter and setter will not be thread-safe so far.